src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```
@@ -39,6 +48,7 @@
     Grpc.Net.Client implementation-->
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Grpc.Core" Version="2.45.0" />
+    <Compile Remove="NativeHostIntegration/./*.*" />
```

`**` can be used for "everything recursively".

```suggestion
    <Compile Remove="NativeHostIntegration/**" />
```